packages/mdc-list/README.md

```diff
@@ -368,11 +369,10 @@ CSS Class | Description
 > NOTE: `mdc-list-divider` class can be used between list items (example 1) *OR* between two lists (example 2).
 
 > NOTE: the difference between selected and activated states:
-
-* *Selected* state should be implemented on the `.mdc-list-item` when it is likely to change soon. Eg., selecting one or more photos to share in Google Photos.
-* Multiple items can be selected at the same time when using the *selected* state.
-* *Activated* state is similar to selected state, however should only be implemented once within a specific list.
-* *Activated* state is more permanent than selected state, and will **NOT** change soon relative to the lifetime of the page.
+> * *Selected* state should be implemented on the `.mdc-list-item` when it is likely to frequently change due to user choice. Eg., selecting one or more photos to share in Google Photos.
+> * *Selected* state may apply to one or multiple items in a list at the same time.
```

Comment:
We do not support multiple item selection for the list. Only selection mode available right now is single selection mode via `singleSelection` API.

Select & Active looks and behave similarly. Only semantic difference is "--selected" is likely to change during its life cycle, where as, `--activated` does not change until page is navigated away or refreshed.

For example, list where user needs to make a choice and may change in the same session. But activated is not changed until the next page refresh or navigation.

Reply:
didn't notice that our docs already had a note on multiple selection.

I do not see any business logic related to multiple selection (except in checkbox based list). We may need to fix the docs.

Reply:
This was the only mention I found of multiple selection, so I think this should be sufficient for now.